### ISSUE TITLE
Some question about the test result and training dataset

### ISSUE THREAD

**Issue by RaymondCover** [hqucv/siamban#4]
Thanks for your work, I meet some probelm. 
Frist, I test on python3.6 and 0.4.1. I can get only 0.298EAO on VOT19 by using the model_zoo model, Accuracy=0.596, Robustness=0.401, EAO=0.298. I also tried to test it on VOT18, but I can not get the same EAO in the paper. I want to konw   would hp_seach be useful for this problem? Or it is just the problem casued by different test envirorment.
Second, I find that you used 100K img_pairs, but in SiamRPN++ and SiamFC++,they use 60K img_pairs. Does this anchor -free method needs more training data?
Lastly, I found that the ablation study about Sample Label Assignment in the paper only used AUC to evaluate，Did Sample Label Assignment influence the robustness？
Thanks for your help

**Comment by zeduchen** [comment]
First, perhaps it is caused by the test environment, you can try to create a new environment.
Second, no. You can try to use 60k image pairs.
Lastly, a comparison on the OTB100 can better reflect their differences, but the OTB100 has no robust comparison.

**Comment by RaymondCover** [comment]
@zeduchen Now I change my environment and get the EAO in the paper ，really thanks for your help

**Comment by Tangzhaotz** [comment]
> @zeduchen Now I change my environment and get the EAO in the paper ，really thanks for your help

are you use 100K img_pairs?
